application/pom.xml

```diff
@@ -58,6 +58,11 @@
             <artifactId>extensions-core</artifactId>
         </dependency>
         <dependency>
+            <groupId>org.thingsboard.extensions</groupId>
+            <artifactId>extension-spark-computation</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
```

Comment:
Maven dependency plugin takes care of this dependency when shipped as debian/jar/exe, but in case of local run using maven-spring-boot-plugin it (other extensions as well) is not added to classpath causing ClassNotFoundException while compiling Dynamically generated classes which depend on this module.